playbooks/ansible-test-network-integration-base/files/bootstrap-eos.yaml

```diff
@@ -1,8 +1,20 @@
 ---
 - hosts: appliance
-  gather_facts: false
   connection: network_cli
   tasks:
+    - name: lookup SSH public key (FIPS)
+      set_fact:
+        _ssh_key: "{{ lookup('file', '~/.ssh/id_ecdsa.pub') }}"
+      when: "'libssh' in zuul.job"
```

Comment:
rather then checking for libssh, we could just default to ecdsa keys for eos now